### DISCUSSION TITLE
玲珑好像不支持一次安装多个应用

### DISCUSSION THREAD

**Discussion by Lichenkass** [linuxdeepin/developer-center#5124] [General | 常规讨论]
玲珑好像不支持一次安装多个应用，类似

> sudo apt install nvidia-driver nvidia-smi nvidia-settings

这种,卸载和更新也不支持

**Comment by BLumia** [comment]
确实目前还不支持这样。cc @black-desk

**Reply by black-desk** [reply]
是的 我知道
